superset-frontend/src/views/CRUD/data/dataset/AddDataset/Header/index.tsx

```diff
@@ -17,7 +17,85 @@
  * under the License.
  */
 import React from 'react';
+import { t, styled, css, SupersetTheme } from '@superset-ui/core';
+import { PageHeaderWithActions } from 'src/components/PageHeaderWithActions';
+import Button from 'src/components/Button';
+import Icons from 'src/components/Icons';
+import { Menu } from 'src/components/Menu';
+import { TooltipPlacement } from 'antd/es/tooltip';
+
+const editableTitleProps = {
+  title: '',
+  placeholder: 'Add the name of the dataset',
```

Comment:
I think all the text in this file needs to be localized

Reply:
Good call, fixed in [`this commit`](https://github.com/apache/superset/pull/21189/commits/890af00638acaa42d9f7d35bd86f3ad573739122).